manifests/b/Badlion/BadlionClient/3.2.0/Badlion.BadlionClient.installer.yaml

```
@@ -1,11 +1,23 @@
-# Created using wingetcreate
+# Created using wingetcreate, modified manually
 
 PackageIdentifier: Badlion.BadlionClient
 PackageVersion: 3.2.0
+# MinimumOSVersion: 
+Platform:
+- Windows.Desktop
+InstallModes:
+- interactive
+- silent
 Installers:
-- Architecture: x64
+- InstallerLocale: en-US
+  Architecture: x64
   InstallerType: nullsoft
+  Scope: user
   InstallerUrl: https://client-updates-cdn77.badlion.net/Badlion%20Client%20Setup%203.2.0.exe
   InstallerSha256: C43392F5C2417BB28F35A19F2DABC8DFB94DA070A647641216625C511F21F3EF
+  # InstallerSwitches:
+  #  Custom: 
+  # ProductCode:
+  UpgradeBehavior: install
 ManifestType: installer
-ManifestVersion: 1.0.0
+ManifestVersion: 1.0.1
```

```suggestion
ManifestVersion: 1.0.0

```
This key must have the value "1.0.0". The Microsoft community package repository validation pipelines also use this value to determine appropriate validation rules when evaluating this file.